evergreen.yml

```
@@ -1092,8 +1094,8 @@ functions:
           ./resources/run_adf.sh start && cargo run --bin data_loader &&
 
           # Run integration tests with ubuntu
-          cargo test integration -- --skip test_driver_log_level --nocapture &&
-          cargo test --test connection_tests integration::test_driver_log_level -- --nocapture &&
+          cargo test integration -- --nocapture &&
+          cargo test --test connection_tests integration::test_driver_log_level --features evergreen_tests -- --nocapture &&
```

If you accept my suggestion about using `cfg_attr`, I believe all integration tests can be run in one step with the following (it works locally!)
```suggestion
          cargo test integration -- features evergreen_tests -- --nocapture &&
```

---

Thank you for the suggestion, I appreciate you taking the time to try to make things more concise.
The reason this is broken in 2 lines is by design.
Tests run in parallel and the logger test can only run by itself.
Otherwise the log files will be polluted and/or held up by the other tests.

I will add a comment so that it is clear for everyone going forward ;)

---

We could specify `--test-threads=1`?

---

I think there is value in running all other tests in parallel and singling this one out.
Why penalize all others tests just because of one specific one is how I see it.